PlayCover/Views/App Views/StoreAppView.swift

```diff
@@ -37,8 +37,16 @@ struct StoreAppView: View {
                 if downloadVM.inProgress {
                     Log.shared.error(PlayCoverError.waitDownload)
                 } else {
-                    DownloadApp(url: url, app: app,
-                                warning: warningMessage).start()
+                    let redirectHandler = RedirectHandler() // checking page redirect
+                    redirectHandler.redirectCatch(from: url)
+                    redirectHandler.waitForAllTasksToComplete()
```

Comment:
I think it's better to hide these implementation details (`redirectCatch` and `waitForAllTasksToComplete`) inside the `RedirectHandler` class. And the `finalURL` should have something like public getter and private setter? 

I prefer styles like
```
let landingUrl = RedirectHandler.catch(from: url)
```

And also hide the check for `"drive"` inside the class. i.e. return the original URL if there are no redirections.

Reply:
Done, now the variable finalURL is private with getter and setter, initialized during class construction with the "reale" url so the "drive" check is no more necessary since if finalURL is not edited getter will return the "real" one